src/main/java/com/example/onlinebookstore/OnlineBookStoreApplication.java

```diff
@@ -1,13 +1,30 @@
 package com.example.onlinebookstore;
 
+import com.example.onlinebookstore.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
+    @Autowired
+    private BookService bookService;
 
     public static void main(String[] args) {
         SpringApplication.run(OnlineBookStoreApplication.class, args);
     }
 
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return new CommandLineRunner() {
+            @Override
+            public void run(String... args) throws Exception {
+
+                System.out.println(bookService.getAll());
+            }
+        };
+    }
```

Comment:
Your CommanLineRunner should set all Book entity fields, and then test both methods from your service